src/events/migrations/0029_merge_20200902_1718.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.0.3 on 2020-09-02 09:18
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0028_auto_20200902_0343'),
+        ('events', '0028_auto_20200902_1549'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Can we squash the two migrations?

Reply:
There are conflicts on 0028. However, the first 0028 is already applied in prod. So I'll just make the latter 0028 0029, and kill the 0029_merge (I think that should work)